runtime/moonbase/src/lib.rs

```diff
@@ -307,6 +319,20 @@ impl FeeCalculator for FixedGasPrice {
 	}
 }
 
+/// Parameterized slow adjusting fee updated based on
+/// https://w3f-research.readthedocs.io/en/latest/polkadot/overview/2-token-economics.html
```

Comment:
```suggestion
/// https://w3f-research.readthedocs.io/en/latest/polkadot/overview/2-token-economics.html#-2.-slow-adjusting-mechanism
```

This might be too long for formatting, but I think it's a more directly relevant place to link.

Reply:
Any idea how to squelch `editorconfig`?

Reply:
`git rm .editorconfig`

But seriously, I don't know if that's possible sadly.

Reply:
here it is: https://github.com/editorconfig-checker/editorconfig-checker#excluding

Reply:
518ed4d